docs/mkdocs/docs/api/basic_json/end_pos.md

```diff
@@ -0,0 +1,67 @@
+# <small>nlohmann::basic_json::</small>end_pos
+
+```cpp
+#if JSON_DIAGNOSTIC_POSITIONS
+constexpr std::size_t end_pos() const noexcept;
+#endif
+```
+
+Returns the position after the last character of the JSON string the value was parsed from.
```

Comment:
parsed from, if the value was parsed from a string.  If this value was constructed in some other way, it will return `std::string::npos`.

Reply:
At some point, I need to be more explicit what "parsed from" means. As it is not just `parse`, but also `operator ""_json`, and maybe also `sax_parse` (need to look into this).

Reply:
Done in 37bb4d9c5d504ff0e8a9cd00cb33c84009ebcf95